ironweb.d.ts

```diff
@@ -272,7 +273,7 @@ export const codec: Codec;
 export const user: User;
 export const group: Group;
 
-export function initialize(jwtCallback: JWTCallback, passcodeCallback: PasscodeCallback): Promise<SDKInitializationResult>;
+export function initialize(jwtCallback: JWTCallback, passcodeCallback: PasscodeCallback, needsRotation: boolean): Promise<SDKInitializationResult>;
```

Comment:
I don't recall what we decided to do with automatically rotating the users private key if it was marked as such after initialize. @clintfred do you recall? If we make it automatic then there's no need for this parameter. If we're making the caller manually invoke rotation then we have to make this new argument optional (with a default of false) in order to not break backward compatibility.

Reply:
We are not automatically doing any key rotation. We do want to be able _notify_ that there are private keys that are marked for rotation at initialization time and then make it reasonably easy to do those rotations. In ironoxide there are two initialize calls, one that will do the rotation checks and one that will not. https://github.com/IronCoreLabs/ironoxide/blob/9f5c7e35283803ae77d5b67b0d07c8d3111a46ee/src/lib.rs#L159

Actually doing the rotation is going to require some sort of call. I think it may be wise to eventually provide a `rotateAll` type of call that will rotate all the keys (both user and group) that are marked for rotation, but it's not strictly necessary of course.

Reply:
Ok, if that's the case then we should remove this `needsRotation` argument here. As long as the value is exposed back to users on the post `initialize` call they can then call `SDK.user.rotatePrivateKey` manually to do the rotation on an as-needed basis.